packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -907,6 +907,35 @@ Expected response body:
 }
 ```
 
+### Azure_ResourceManager_Models_Resources_AvailableOperations_list
+
+- Endpoint: `get https://management.azure.com`
+
+  Resource GET operation.
+  Expected path: /providers/Azure.ResourceManager.Models.Resources/operations
+  Expected query parameter: api-version=2023-12-01-preview
+
+  Expected response body:
+
+  ```json
+  {
+    "value": [
+      {
+        "name": "Microsoft.Compute/virtualMachines/write",
+        "isDataAction": true,
```

Comment:
nit, create virtual machine seems not dataAction?

Reply:
Changed `"isDataAction": true` to `"isDataAction": false`.